add Github action (#2)
JohnTwiiX/Conduit_Container: README.md

   // -d - will run the containers in the background
   ```

## Automation

This project has a workflow. This connects to the ssh server pulled or cloned the Conduit project. Variables for the Django configuration are passed automatically, you just have to set the appropriate variables and secrets

> 1. set Github secrets and variables

Reviewer: ```suggestion
### Configure Github secrets and variables
```

>    - go to `setting -> security -> Secrets and variables -> actions`

Reviewer: ```suggestion
Follow these steps:

1. Go to `settings -> security -> Secrets and variables -> actions`
2. select X, provide value, press save

```

     | Key               | description                              | secret/variable |
     | ----------------- | ---------------------------------------- | --------------- |
     | `PAT`             | the personal access token                | secret          |
     | `PROJECT_PATH`    | the path to the project                  | secret          |
     | `SERVER_IP`       | the ip of your server                    | secret          |
     | `SSH_PRIVATE_KEY` | the ssh private key to connect to server | secret          |
     | `USER`            | the user on the server                   | secret          |
     | `API_URL`         | the url for the backend                  | variable        |

> 1. create the personal access token with this [dokumentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic)

Reviewer: ```suggestion
1. create the personal access token with this [documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-personal-access-token-classic)
```

Reviewer: Explain additionally why a PAT is required in that place
